lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Settings.hs

```diff
@@ -76,3 +83,48 @@ spec = describe "SHELLEY_SETTINGS" $ do
                     (`shouldBe` (either (const (error "no")) id $ fromText
                         @PoolMetadataSource uri))
                 ]
+
+    it "SETTINGS_02 - Changing pool_metadata_source re-syncs metadata" $ \ctx -> do
+        let toNone = "none"
+            toDirect = "direct"
+            getMetadata = fmap (view #metadata) . snd <$> unsafeRequest
+                @[ApiStakePool] ctx (Link.listStakePools arbitraryStake) Empty
+            delay = 500 * 1000
+            timeout = 300
```

Comment:
This seems arbitrarily long?

Reply:
It was temporary to see if it possibly passes on Hydra... but hasn't -> https://github.com/input-output-hk/cardano-wallet/pull/2225#issuecomment-710070717